x-pack/plugins/ml/public/components/loading_indicator/loading_indicator.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+import './styles/main.less';
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export function LoadingIndicator({ height }) {
```

Comment:
Would be good to use `EuiLoadingSpinner` in place of the old spinner.

Reply:
Took a note in the follow up issue: https://github.com/elastic/kibana/issues/22626